azalea/examples/nearest_entity.rs

```diff
@@ -0,0 +1,73 @@
+use azalea::nearest_entity::EntityFinder;
+use azalea::ClientBuilder;
+use azalea::{Bot, LookAtEvent};
+use azalea_client::Account;
+use azalea_entity::metadata::{Item, ItemItem};
+use azalea_entity::{EyeHeight, Position};
+use bevy_app::{FixedUpdate, Plugin};
+use bevy_ecs::{
+    prelude::{Entity, EventWriter},
+    query::With,
+    system::Query,
+};
+
+#[tokio::main]
+async fn main() {
+    let account = Account::offline("bot");
+
+    ClientBuilder::new()
+        .add_plugins(LookAtStuffPlugin)
+        .start(account, "localhost:34071")
```

Comment:
I'm guessing this is because you opened to LAN and didn't change the port, but in 1.20 you can just specify the port to be 25565 when opening to LAN.